documentation/mDNS_Support.md

```diff
@@ -20,17 +20,18 @@ Our sample file `physical-web-url.service` looks like this:
 <!DOCTYPE service-group SYSTEM "avahi-service.dtd">
 
 <service-group>
-  <name replace-wildcards="yes">http://www.mycompany.com/xyz.html</name>
+  <name replace-wildcards="yes">Physical Web on %h</name>
   <service>
-    <host-name>www.mycompany.com</host-name>
-    <type>_http._tcp</type>
+    <type>_urlbookmark._tcp</type>
+    <host-name>local</host-name>
```

Comment:
Don't think you need to set host-name at all.
Using this line all your FQDN are equal to local, this seems not correct.

<pre>man avahi.service</pre>
&lt;host-name&gt;  The  host name of the host that provides this service. This should be a host that is resolvable by multicast or unicast DNS. Please note that you need to specify a fully-qualified domain name (FQDN) here, i.e. .local is not appended implicitly! The  host  name  doesn't need to be part of the domain specified in &lt;domain-name&gt;. See avahi.hosts(5) for more information how to publish additional host name mappings.